src/pages/Register/Register.css

```diff
@@ -1,25 +1,40 @@
-/* body {
```

Comment:
I understand that the background and the fonts are not used anymore?

Reply:
We will set the fonts for the whole app at app.css (or similar) to not repeat it in all components